### DISCUSSION TITLE
Stealth Mode

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#1108] [General]
More and more people use the gopro as a vlogging camera and don't want to catch attention by the display on the back of the camera (which only shuts off after 1 minute). That's why early gopros and the sony x3000 were so popular.

It's not possible to just put a black screen with in white: big battery percentage indicator on the right and recording time at the left? Saves battery and is more handy for different scenarios. Great option to have anyway.

**Comment by dnewman-gpsw** [comment]
You can shutdown the rear LCD with commands ```oDO``` or ```$DLCD=1``` (disable LCD).   Weirdly these are don't have an undo i.e.  ```$DLCD=0``` doesn't current turn the LCD back on.   But it will save power.  Using permanent ```*FAST=1*DLCD=1```, the rear LCD is only on for a few seconds.

**Reply by lyricsdude** [reply]
thank you, i will experiment with this :)

**Reply by lyricsdude** [reply]
would be perfect if I can set the time to 5 seconds till the screen turns off. then with a tap i can turn it on again for 5 seconds to see if it's still recording at anytime.
